entries/workflow.md

```diff
@@ -8,33 +8,19 @@ in more detail in the [GitHub
 Flow](https://docs.github.com/en/get-started/quickstart/github-flow)
 page.
 
-There are some scripts to help automate some of the tasks of working on
-the project repositories. They are all found in the `tools/` folder. All
-of them require opening a Terminal, and how you do that depends on the
+We use some tools to help automate some of the tasks of working on
```

Comment:
Could it be a bit more specific what "some of the tasks of working on the project repos" are? :)